.github/workflows/upgrade-e2e.yml

```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   upgrade-e2e:
     name: Latest Release to Latest Version
+    if: github.base_ref == 'devel'
```

Comment:
Use instead?
```
on:
  pull_request:
    branches:
      - devel
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches

Reply:
Just to clarify something, this will skip the job entirely, which makes more sense than the `if` since with the `if` solution the job would appear to have passed but it didnt actually do anything.

This necessitates that this job be defined as "required" only in the devel branch (which also makes sense overall), otherwise it wont let us merge if the job was skipped